chromium_src/chrome/browser/ssl/https_upgrades_navigation_throttle.cc

```diff
@@ -30,81 +24,28 @@ bool IsTor(content::NavigationHandle* handle) {
   return profile->IsTor();
 }
 
+bool NormalWindowHttpsOnly(content::NavigationHandle* handle,
```

Comment:
Same as below. Also please add tests that cover the behavior that previously caused infinite redirects